server/config.ts

```diff
@@ -83,4 +83,17 @@ export default {
     },
   },
   domain: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
+  serviceSignifier: get<'path' | 'domain' | 'approved-premises-only' | 'temporary-accommodation-only'>(
+    'SERVICE_SIGNIFIER',
+    'path',
+    requiredInProduction,
+  ) as 'path' | 'domain' | 'approved-premises-only' | 'temporary-accommodation-only',
```

Comment:
I wonder if we should have default service signifiers, so locally we're using the path and then in 'prod-like' envs we're using the domain